packages/quantic/force-app/main/default/lwc/quanticSmartSnippetSuggestions/e2e/pageObject.ts

```diff
@@ -0,0 +1,115 @@
+import type {Locator, Page, Request} from '@playwright/test';
+import {
+  isUaCustomEvent,
+  isUaClickEvent,
+} from '../../../../../../playwright/utils/requests';
+
+export class SmartSnippetSuggestionsObject {
+  constructor(public page: Page) {
+    this.page = page;
+  }
+
+  get smartSnippetSuggestions(): Locator {
+    return this.page.locator('c-quantic-smart-snippet-suggestions');
+  }
+
+  get smartSnippetSuggestionHeading(): Locator {
+    return this.page.locator(
+      'lightning-accordion-section .slds-accordion__summary-heading'
+    );
+  }
+
+  get smartSnippetSuggestionSourceUri(): Locator {
+    return this.page.locator('[data-testid="smart-snippet__source-uri"]');
+  }
+
+  get smartSnippetSuggestionSourceTitle(): Locator {
+    return this.page.locator('[data-testid="smart-snippet__source-title"]');
```

Comment:
And for all others too.

```suggestion
    return this.page.getByTestId('smart-snippet__source-title');
```